### PR TITLE
Edit journal entity

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR add the option of editing the entity of transactions in the posting journal.

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles @sfount could you review this PR 
I don't know how to fix this `$rootScope:inprog` error

File: server/controllers/finance/journal/index.js [hunk 1]
```diff
@@ -307,15 +307,18 @@ function editTransaction(req, res, next) {
...
         transaction.addQuery(INSERT_JOURNAL_ROW, [row]);
       });
 
       return transformColumns(rowsChanged, false);
     })
     .then((result) => {
-      _.each(result, (row, uid) => transaction.addQuery(UPDATE_JOURNAL_ROW, [row, db.bid(uid)]));
+      _.each(result, (row, uid) => {
+        db.convert(row, ['entity_uuid']);
+        return transaction.addQuery(UPDATE_JOURNAL_ROW, [row, db.bid(uid)]);
```
Comment (jniles) [inline server/controllers/finance/journal/index.js]: You do not need this `return` here.

File: server/controllers/finance/journal/index.js [hunk 2]
```diff
@@ -307,15 +307,18 @@ function editTransaction(req, res, next) {
   transformColumns(rowsAdded, true)
     .then((result) => {
       result.forEach((row) => {
-        db.convert(row, ['uuid', 'record_uuid']);
+        db.convert(row, ['uuid', 'record_uuid', 'entity_uuid']);
```
Comment (jniles) [inline server/controllers/finance/journal/index.js:310]: 👍

File: client/src/js/services/grid/TransactionService.js [hunk 3]
```diff
@@ -207,6 +207,26 @@ function TransactionService($timeout, util, uiGridConstants, bhConstants, Notify
...
   /**
+   * @function editCellGrid
+   *
+   * @description
+   * Edit cell value for custom template
+   *
+   * @public
+   *
+   */
+  Transactions.prototype.editCellGrid = function editCellGrid(rowEntity, field, value) {
```
Comment (jniles) [inline client/src/js/services/grid/TransactionService.js]: Can you make this function the same function as `editCell`?  It looks duplicated.  And if it must be a different function, can this just call `editCell` internally?

**Comment by ghost** [comment]
:umbrella: The latest upstream changes (presumably 2b2435d85f0d45388888a8874134fc528828d8e6) made this pull request unmergeable. Please resolve the merge conflicts.

**Comment by jniles** [comment]
@mbayopanda, whoops.... sorry about that :(

**Comment by jniles** [comment]
@kwilu try

**Comment by ghost** [comment]
:hourglass: Trying commit 0479fd8c53e6d03ef11aa757dbdd6f02d6264c8c with merge 2a907dfd97cbb95e42c7d2ae2120a67fa1af1945...

**Comment by mbayopanda** [comment]
@kwilu try

**Comment by ghost** [comment]
@mbayopanda: :key: Insufficient privileges: Not in reviewers

**Comment by ghost** [comment]
:hourglass: Trying commit 0479fd8c53e6d03ef11aa757dbdd6f02d6264c8c with merge 51c2d94ea889707f7e17277ab1ce468c7dd2f401...

**Comment by jniles** [comment]
LGTM!

**Pull request merged**